src/libs/ReportUtils.js

```diff
@@ -845,6 +846,16 @@ function getPersonalDetailsForLogin(login) {
     );
 }
 
+/**
+ * Gets the accountID for a login by looking in the ONYXKEYS.PERSONAL_DETAILS Onyx key (stored in the local variable, allPersonalDetails). If it doesn't exist in Onyx,
+ * then an empty string is returned.
+ * @param {String} login
+ * @returns {String}
+ */
+function getAccountIDForLogin(login) {
```

Comment:
@cloudpresser @puneetlath
This method no longer works after `accountID` migration because we have `accountID` as they and not `login` anymore. This is causing the following issues: 
- https://github.com/Expensify/App/pull/20746#issuecomment-1595174744
- https://github.com/Expensify/App/pull/20746#issuecomment-1595188280

Reply:
Issue reported here: https://github.com/Expensify/App/issues/20934